playbooks/roles/cluster/files/cluster_provision_survey.json

```diff
@@ -0,0 +1,85 @@
+{
+    "description": "",
+    "name": "",
+    "spec": [
+      {
+        "question_description": "Desired name of Openshift cluster",
+        "min": 0,
+        "default": "",
+        "max": 16,
+        "required": true,
+        "choices": "",
+        "variable": "oo_clusterid",
+        "question_name": "Cluster Name",
+        "type": "text"
+      },
+      {
+        "question_description": "Region you wish to provision the cluster in",
+        "min": null,
+        "default": "",
+        "max": null,
+        "required": true,
+        "choices": "us-east-1\neu-west-1\neu-west-2",
+        "variable": "oo_sublocation",
+        "question_name": "AWS Region",
+        "type": "multiplechoice"
+      },
+      {
+        "question_description": "Flag for provisioning logging stack",
+        "min": null,
+        "default": "",
+        "max": null,
+        "required": true,
+        "choices": "true\nfalse",
+        "new_question": true,
+        "variable": "survey_provision_logging",
+        "question_name": "Provision Logging",
+        "type": "multiplechoice"
+      },
+      {
+        "question_description": "Token for Credential repo visit: https://github.com/<credential-repo>/blob/master/files/concurrent_provisioning_vars.yml.j2 and click raw to find token",
```

Comment:
Is there anything sensitive in the provisioning vars template? If not we could look at publicly hosting it somewhere and not ask for a token. Not sure how it would work for Delorean as these jobs are automated and would have no way of dynamically setting the token.